scripts/create_table.sql

```diff
@@ -0,0 +1,26 @@
+-- create doctors, patients, and appointments tables
+CREATE TABLE doctors (
+  id SERIAL PRIMARY KEY,
+  surname TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  specialty TEXT
+);
+
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  surname TEXT,
+  given_name TEXT,
+  gender TEXT,
+  born_on DATE,
+  insurance TEXT
+);
+
+CREATE TABLE appointments (
+  id SERIAL PRIMARY KEY,
+  comment TEXT,
+  app_time TIME,
+  app_date DATE,
+  insurance TEXT
+);
```

Comment:
your `appointments` table is missing a reference to the `doctors` and `patients` tables

```sql
CREATE TABLE appointments(
  date TEXT, time TEXT, comment TEXT,
  doctor_id INTEGER REFERENCES doctors(id),
  patient_id INTEGER REFERENCES patients(id)
);
```

Also, `appointments` table should not have a `insurance` column